lib/router/config-router.js

```diff
@@ -110,7 +110,10 @@ function ConfigRouter(conf, schedulerRegistry) {
     this.disabledProtocols.push('websocket');
   }
 
-  this.reconnect = !conf.getValues('disable_reconnect').enabled;
+  this.reconnect = false;
```

Comment:
You can mark a param in the config as optional and provide a default there. Which is what we do with `enabled`. But I'm fighting for config-file backwards compatibility. I don't want the meaning of someone's

```
disable_reconnect;
```

to change in the next version. I just want to internally flip form opt-in to opt-out.